Piquet.cpp

```diff
@@ -1,12 +1,19 @@
 #include <iostream>
 #include "Piquet.h"
+#include <cmath>
+#include <limits>
+
+bool is_equal(double x, double y) {
+    return std::fabs(x - y) < std::numeric_limits<double>::epsilon();
```

Comment:
`std::math`
In C++, `std::abs` is overloaded for both signed integer and floating point types. `std::fabs` only deals with floating point types (pre C++11). Note that the `std::` is important; the C function `::abs` that is commonly available for legacy reasons will only handle `int`!